sdk/communication/communication-phone-numbers/test/README.md

```diff
@@ -11,7 +11,7 @@ The Azure resource that is used by the tests in this project is:
 To run the live tests, you will need to set the below environment variables:
 
 - `TEST_MODE`: Should have `live` assigned if you want to run live without recording. Assign `record` to run live with recording.
-- `COMMUNICATION_CONNECTION_STRING`: The primary connection string of the Communication Services resource in your account.
+- `COMMUNICATION_LIVETEST_STATIC_CONNECTION_STRING`: The primary connection string of the Communication Services resource in your account.
```

Comment:
Oh interesting I guess this read me was always incorrect and this is correcting it.